src/app/pages/vm/dockervm-wizard/dockervm-wizard.component.ts

```diff
@@ -57,7 +57,7 @@ export class DockerVMWizardComponent {
         name : 'name',
         placeholder :  helptext.docker_vm_placeholder,
         tooltip : helptext.docker_vm_tooltip,
-        validation : helptext.docker_vm_validation,
+        validation : [Validators.required,Validators.pattern('^[a-zA-Z0-9_]*$')],
```

Comment:
you should change the validators on in the helptext file rather than directly in here